stylesheet.css

```diff
@@ -0,0 +1,69 @@
+body {
+    background-color: #f1f1f1;
+    font-family: calibri;
+    color:#5f5f5f;
+}
+
+header {
+  background: #5f5f5f;
+    padding: 1em;
```

Comment:
this file is also having indentation and white space issues. Auto format will serve you well here as well. Might ask to install extension to support css files... just install most popular one...